helm-charts/nginx_values.yaml

```diff
@@ -2,14 +2,30 @@
 ## Ref: https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md
 ##
 controller:
-  ingressClass: nginx
+  ingressClass: nginx-controller
+  image:
+    repository: us.gcr.io/k8s-artifacts-prod/ingress-nginx/controller
```

Comment:
please remove this if we want to have upgrades in the future, with this we hardcode the version/image and when upgrading the chart it will need a change here as well

Reply:
I removed the image version and forgot this, thanks :)